tests/integrations/test_dbm.py

```diff
@@ -31,12 +31,56 @@ def setup_trace_payload(self):
                 weblog.get("/dbm", params={"integration": "mysqli"}),
             ]
 
-    def test_trace_payload(self):
+    def find_db_spans(self):
```

Comment:
Hey @cbeauchesne, so in this case Im not sure how to apply said changes, I've tried for a couple of hours but the outcome is that instead of being able to find my spans for each request like in the screenshot below:
![image](https://github.com/DataDog/system-tests/assets/28125428/90ede4af-738b-4cac-91c4-7337e87c0a20)

I end up breaking the test and having it only go trough one request, this is also part of why I stopped using `get_spans` because I needed to iterate trough each span to find the right one, before the test was flaky because it would match against any span and not check the tags so not sure how to accomplish that based on your latest recommendations:
![image](https://github.com/DataDog/system-tests/assets/28125428/e8c998ab-8ace-4228-961e-0f27d13a8dcb)

The above is how the test logs end up looking like so if you have some time for us to sync on this tomorrow that would be great!

Reply:
I've sent you an invite